docs/dataset/advanced-usage/extract-text-metadata.md

```diff
@@ -47,7 +47,7 @@ to get started with Automatic Metadata Extraction for Text.
 !!! example
 
     <figure markdown>
-    ![Hydrated Question](../../assets/images/hydrated-text-example.png)
+    ![Hydrated Question](../../assets/images/hydrated-text-example-light.png)
```

Comment:
Showing light image in dark mode:
<img width="1306" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/413577d2-8fe8-480b-b2f6-0555c4373e8f">

You need to include both light and dark images and use the following format:
```
![Precision Image](../assets/images/metrics-precision-light.svg#only-light)
![Precision Image](../assets/images/metrics-precision-dark.svg#only-dark)
```

Also, we need another copy of gifs for light mode

Reply:
Added screenshots/gifs for both dark and lightmode.

Reply:
Ah sorry for the unclear instruction! Feel free to message me if you have any questions :)